sources/data-node/api/v1/trading_data.proto

```diff
@@ -1152,15 +1152,33 @@ message LastBlockHeightResponse {
 
 // A request to get reward details for a party
 message GetRewardDetailsRequest {
-  string party_id = 1;
+  string party_id = 1 [(validator.field) = {string_not_empty : true }];
 }
 
-// The return message with reward details in
-message GetRewardDetailsResponse {
+// Details for a single reward payment
+message RewardDetails {
```

Comment:
Just a question.. Should this be part of the API or part of general Vega types?

Reply:
Updated, thanks.